packages/vuetify/src/components/VTooltip/VTooltip.tsx

```diff
@@ -0,0 +1,108 @@
+// Styles
+import './VTooltip.sass'
+
+// Components
+import { VOverlay } from '@/components/VOverlay'
+
+// Composables
+import { useProxiedModel } from '@/composables/proxiedModel'
+import { makeTransitionProps } from '@/composables/transition'
+
+// Utilities
+import { computed } from 'vue'
+import { defineComponent, getUid } from '@/util'
+
+// Types
+import type { PropType } from 'vue'
+import type { StrategyProps } from '@/components/VOverlay/positionStrategies'
+
+export default defineComponent({
+  name: 'VTooltip',
+
+  props: {
+    id: String,
+    modelValue: Boolean,
+    text: String,
+
+    anchor: {
+      type: String as PropType<StrategyProps['anchor']>,
+      default: 'end',
+    },
+    origin: {
+      type: String as PropType<StrategyProps['origin']>,
+      default: 'auto',
+    },
+
+    ...makeTransitionProps({
+      transition: false,
+    } as const),
+  },
+
+  emits: {
+    'update:modelValue': (value: boolean) => true,
+  },
+
+  setup (props, { attrs, slots }) {
+    const isActive = useProxiedModel(props, 'modelValue')
+
+    const uid = getUid()
+
+    const id = computed(() => props.id || `v-tooltip-${uid}`)
+
+    const anchor = computed(() => {
+      return props.anchor.split(' ').length > 1
+        ? props.anchor
+        : props.anchor + ' center' as StrategyProps['anchor']
+    })
+
+    const origin = computed(() => {
+      return (
+        props.origin === 'auto' ||
+        props.origin === 'overlap' ||
+        props.origin.split(' ').length > 1 ||
+        props.anchor.split(' ').length > 1
+      ) ? props.origin
+        : props.origin + ' center' as StrategyProps['origin']
+    })
```

Comment:
Is this different logic for these props than elsewhere?

Reply:
Yes, on menus `anchor="top"` means top-left, tooltips should be center by default instead.

Reply:
v2 did this by overriding a bunch of methods, this way is far simpler and still allows anchoring on corners if you want to.